src/styles/_variables.scss

```diff
@@ -0,0 +1,20 @@
+//landing page header
+$mars-header-background: #ff9d1e;
+$mission-header-background: #ffebd1;
+$header-font: #dd361c;
+
+//base colors
+$color-base: #212121;
+$color-white: #ffffff;
+$color-primary: #105bd8;
+$color-primary-darker: #0b3d91;
+$color-primary-darkest: #061f4a;
+$color-grey-light: #aeb0b5;
+
+//landing page buttons
+$button-color-one: #02bfe7;
+$button-color-two: #4aa564;
+$button-color-three: #4c2c92;
+$button-color-four: #f9e0de;
+$button-background-color: #ffebd1;
+$button-background-hover: #f9aa43;
```

Comment:
These names aren't very descriptive:
1. What do the four different button colours refer to (usage-wise)?
2. If the four button colours aren't the background, what are they?

Please name these more descriptively 😊